tests/test_observability.py

```
@@ -0,0 +1,137 @@
+    monkeypatch.setattr("shared.observability.logger", mock_logger)
+
+    safe_task_log(
+        logging.INFO,
+        "日常检查完成",
+        group_name="我的关注",
+        group_id="group_1",
+        unknown_field="secret_val",
+        instance="",
+    )
+
+    level, message = mock_logger.log.call_args[0]
```

Ruff (0.16.1)

[warning] Unpacked variable `level` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)